homeassistant/components/overkiz/const.py

```diff
@@ -97,3 +116,42 @@
     OverkizCommandParam.SFC: "sfc",
     OverkizCommandParam.UPS: "ups",
 }
+
+OVERKIZ_UNIT_TO_HA: dict[str, str] = {
+    MeasuredValueType.ABSOLUTE_VALUE: "",
+    MeasuredValueType.ANGLE_IN_DEGREES: DEGREE,
+    MeasuredValueType.ANGULAR_SPEED_IN_DEGREES_PER_SECOND: f"{DEGREE}/{UnitOfTime.SECONDS}",
+    MeasuredValueType.ELECTRICAL_ENERGY_IN_KWH: UnitOfEnergy.KILO_WATT_HOUR,
+    MeasuredValueType.ELECTRICAL_ENERGY_IN_WH: UnitOfEnergy.WATT_HOUR,
+    MeasuredValueType.ELECTRICAL_POWER_IN_KW: UnitOfPower.KILO_WATT,
+    MeasuredValueType.ELECTRICAL_POWER_IN_W: UnitOfPower.WATT,
+    MeasuredValueType.ELECTRIC_CURRENT_IN_AMPERE: UnitOfElectricCurrent.AMPERE,
+    MeasuredValueType.ELECTRIC_CURRENT_IN_MILLI_AMPERE: UnitOfElectricCurrent.MILLIAMPERE,
+    MeasuredValueType.ENERGY_IN_CAL: "cal",
+    MeasuredValueType.ENERGY_IN_KCAL: "kcal",
+    MeasuredValueType.FLOW_IN_LITRE_PER_SECOND: f"{UnitOfVolume.LITERS}/{UnitOfTime.SECONDS}",
```

Comment:
Should this instead be added as a `UnitOfVolumeFlowRate` so it can be converted automatically?

Reply:
About unit conversion, something special should be done except adding a value within an enum? I’ve noticed for example that Unifi integration exposed some sensors in `MB`, but no conversion is proposed into the UI.

Reply:
Unit conversion is based on `device_class`, so maybe the unifi integration sensor doesn't have a device class set...

Reply:
You are right, this one was missing. Thx for the input.

Reply:
About the initial comment: https://github.com/home-assistant/core/pull/88966